docs/addresses.md

```diff
@@ -8,6 +8,7 @@ description: List of Standard Contract Addresses
 |--------------|--------------------------------------------|--------------------------------------------|------------------------------------------------------------------|---------------------------------|
 | [Multicall V3][multicall] | `0xcA11bde05977b3631167028862bE2a173976CA11` | - | [Mainnet][multicall-verify-mainnet] | [Multicall3.sol][multicall-source] |
 | [Wrapped ROSE][wrose-dapp] | `0x8Bc2B030b299964eEfb5e1e0b36991352E56D2D3` | `0xB759a0fbc1dA517aF257D5Cf039aB4D86dFB3b94` | [Mainnet][wrose-verify-mainnet], [Testnet][wrose-verify-testnet] | [WrappedROSE.sol][wrose-source] |
+| [Celer IM Executor][message-executor] | - | `0x9C850D230FFFaCEf1E2D1741a00080856630e455` | [Testnet][message-executor-testnet] | [Message Executor][message-executor-source] |
```

Comment:
Where's the Mainnet address?

Also, since this is not a contract address, it doesn't really make sense to put anything into the `Verify` column IMO.

Reply:
> Where's the Mainnet address?

Fair. Looking.

> Also, since this is not a contract address, it doesn't really make sense to put anything into the Verify column IMO.

Makes sense. I like having a unified page for configurations. I'll suggest a renaming of this page and then add another appropriate section.